modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/add_to_cart/AddToCart.js

```diff
@@ -246,7 +246,7 @@ function AddToCart({
 					allowedQuantities={
 						settings.productConfiguration?.allowedOrderQuantities
 					}
-					disabled={initialDisabled || !account?.id}
+					disabled={buttonDisabled}
```

Comment:
I think we cannot use the same variable as the button because it looks also for "not quantity"- the button is disabled if the input is empty but in this case the input should not be disabled otherwise I'm not able to enter a new value